web/packages/plugins/job/src/JobControl.tsx

```diff
@@ -82,6 +82,7 @@ export const JobControl = (props: {
 }) => {
   const { jobId, document } = props
   const [dataSourceId, documentId] = jobId.split('/', 2)
+  // @ts-ignore
```

Comment:
the type error should be fixed in master branch, and not necessary to add the @ts-ignore